package.json

```diff
@@ -35,49 +35,49 @@
   },
   "devDependencies": {
     "@gravitational/build": "workspace:*",
-    "@storybook/addon-actions": "^8.3.4",
-    "@storybook/addon-controls": "^8.3.4",
-    "@storybook/addon-toolbars": "^8.3.4",
-    "@storybook/components": "^8.3.4",
-    "@storybook/preview-api": "^8.3.4",
-    "@storybook/react": "^8.3.4",
-    "@storybook/react-vite": "^8.3.4",
-    "@storybook/test-runner": "^0.19.1",
-    "@testing-library/jest-dom": "^6.5.0",
+    "@storybook/addon-actions": "^8.4.6",
+    "@storybook/addon-controls": "^8.4.6",
+    "@storybook/addon-toolbars": "^8.4.6",
+    "@storybook/components": "^8.4.6",
+    "@storybook/preview-api": "^8.4.6",
+    "@storybook/react": "^8.4.6",
+    "@storybook/react-vite": "^8.4.6",
+    "@storybook/test-runner": "^0.20.0",
+    "@testing-library/jest-dom": "^6.6.3",
     "@testing-library/react": "^16.0.0",
     "@testing-library/user-event": "^14.5.2",
     "@types/history": "^4.7.11",
-    "@types/jest": "^29.5.13",
-    "@types/node": "^20.16.10",
-    "@types/react": "^18.3.10",
-    "@types/react-dom": "^18.3.0",
+    "@types/jest": "^29.5.14",
+    "@types/node": "^20.17.9",
+    "@types/react": "^18.3.12",
+    "@types/react-dom": "^18.3.1",
     "@types/react-highlight": "^0.12.8",
     "@types/react-router": "^5.1.20",
     "@types/react-router-dom": "^5.1.1",
     "@types/react-transition-group": "^4.4.11",
     "@types/wicg-file-system-access": "^2023.10.5",
     "jest": "^29.7.0",
     "jsdom-testing-mocks": "^1.13.1",
-    "msw": "^2.4.9",
-    "msw-storybook-addon": "^2.0.3",
-    "playwright": "^1.47.2",
-    "prettier": "^3.3.3",
+    "msw": "^2.6.6",
```

Comment:
> @ravicious what do you think if i just removed the `msw` and the other change you pointed out in the last PR to merge in the rest of this group? I can also add the changes you had in the previous

@avatus msw should be fine to go, no? It was just the problem with `BroadcastChannel` and `TransformStream` which I've fixed in the previous PR. That was the easy part, the commit which gets rid of an unnecessary peer dep was a little bit more complex. But I can definitely do this tomorrow.

And then we'd be back to just the change with the styles (cc @bl-nero https://github.com/gravitational/teleport/pull/48268#discussion_r1827767361), so I don't necessarily think we need to be leaving anything out from this PR. Unless there's some new stuff, I haven't looked at the changes yet.

Reply:
> @avatus msw should be fine to go, no? 

yes, i think i just misread the messages from the previous PR